scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+import shutil
+import nibabel as nib
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from dipy.viz import actor, window
```

Comment:
Also, this doesn't seem to work with a recent Dipy. You should use Fury.

Reply:
Ok, should I add the fury dependency in requirement.txt file?

Reply:
Yes please.

Reply:
Ok, fury==0.2.0 added